src/gettingstarted/own-code/connect-services.md

```diff
@@ -0,0 +1,157 @@
+# Import your own code
+
+## Connect to Services
+
+> **Note:** If your application does not require access to services and you left `.platform/services.yaml` blank, feel free to proceed to the next step.
+
+At this point you have configured your application's services as well as its build and deploy process in your `.platform/services.yaml` and `.platform.app.yaml` files. As an example, in your `.platform.app.yaml` file you may have defined a relationship called `database`:
+
+```yaml
+relationships:
+    database: "mysqldb:mysql"
+```
+
+which was configured in `.platform/services.yaml` with
+
+```yaml
+mysqldb:
+    type: mysql:10.2
+    disk: 1024
+```
+
+In order to connect to this service and use it in your application, Platform.sh exposes its credentials in the application container within a base64-encoded JSON `PLATFORM_RELATIONSHIPS` environment variable.
+
+To access this variable you can install a Platform.sh configuration reader library
+
+{% codetabs name="PHP", type="php" -%}
+composer install platformsh/config-reader
+{%- language name="Python", type="py" -%}
+pip install platformshconfig
+{%- language name="Node.js", type="js" -%}
+npm install platformsh-config --save
+{%- language name="Java", type="java" -%}
+// Installing with Maven
+ <dependency>
+     <groupId>sh.platform</groupId>
+     <artifactId>config</artifactId>
+     <version>0.0.1-SNAPSHOT</version>
+ </dependency>
+
+// Installing with Gradle
+compile group: 'sh.platform', name: 'config', version: '0.0.1-SNAPSHOT'
+{%- language name="Go", type="go" -%}
+// In your go.mod file
+github.com/platformsh/config-reader-go/v2 v2.2.2
+{%- endcodetabs %}
```

Comment:
It looks like there is a CLI command option for Go; although I'd love to be able to not need to specify a version and have it pick the latest when you run it.  Maybe this command will do so?

`go mod edit -require=path-to-module`

https://golang.org/cmd/go/#hdr-Edit_go_mod_from_tools_or_scripts